pkgs/development/tools/misc/ccache/default.nix

```diff
@@ -86,7 +86,10 @@ stdenv.mkDerivation (finalAttrs: {
       };
       inherit (unwrappedCC) lib;
       nativeBuildInputs = [ makeWrapper ];
-      buildCommand = ''
+      buildCommand = let
+        # Clang does not yet have targetConfig in passthru.
+        targetPrefix = if stdenv.cc.isGNU then (lib.optionalString (unwrappedCC.targetConfig != null) "${unwrappedCC.targetConfig}-") else stdenv.cc.targetPrefix;
```

Comment:
Unwrapped clang does not have prefixes
`pkgsCross.aarch64-multiplatform.clangStdenv.cc.cc` 

clang uses argv0 to pick the target